tests/spectron/zoom-in-zoom-out.spectron.js

```diff
@@ -1,36 +1,32 @@
-const childProcess = require('child_process');
 const Application = require('./spectronSetup');
 const { isMac } = require('../../js/utils/misc');
 const constants = require('./spectronConstants');
+const robot = require('robotjs');
 
-let robot;
 let configPath;
 
 let app = new Application({});
 
 describe('Tests for Zoom in and Zoom out', () => {
 
     let originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
-    jasmine.DEFAULT_TIMEOUT_INTERVAL = 90000;
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = Application.getTimeOut();
 
     beforeAll((done) => {
-        childProcess.exec(`npm rebuild robotjs --target=${process.version} --build-from-source`, function () {
-            robot = require('robotjs');
-            return app.startApplication().then((startedApp) => {
-                app = startedApp;
-                getConfigPath().then((config) => {
-                    configPath = config;
-                    done();
-                }).catch((err) => {
-                    console.error(`Unable to get user config path error: ${err}`);
-                    expect(err).toBeNull();
-                    done();
-                });
+        return app.startApplication().then((startedApp) => {
+            app = startedApp;
+            getConfigPath().then((config) => {
+                configPath = config;
+                done();
             }).catch((err) => {
-                console.error(`Unable to start application error: ${err}`);
+                console.error(`Unable to get user config path error: ${err}`);
                 expect(err).toBeNull();
                 done();
             });
+        }).catch((err) => {
+            console.error(`Unable to start application error: ${err}`);
+            expect(err).toBeNull();
```

Comment:
why this? when we are in the catch section, we have to expect err to be not null right?